Dockerfile

```diff
@@ -5,6 +5,6 @@ WORKDIR app
 
 ADD . .
 
-EXPOSE 8003
+EXPOSE 8000
```

Comment:
@Gwillison415 There isn't really a "standardized dev approach." `8000` is common (and my personal favorite) to be sure, but `8080`, `3000`, `9000`, and many others are also popular. It's important to keep an eye out for when an individual's preference is cited as best practice. This is more common than you might imagine. The danger is a misunderstanding of technical specifics. In this instance, for example, it is a rather welcome feature of the operating system that we have so many ports to choose from, and helpful at the least to prevent port use conflict errors.

That said this change is harmless and I am happy to accept it, especially since I love ol `8000` so much.

Reply:
Thanks for taking the time to write it up - I don't know if i'd call a specific port of any number a best practice - but i hear what you're saying. I just thought we had established that we'd map to containers of 8000 in the mob fix, i could me wrong. It struck me that the mental work of understanding the difference in host : container mapping would yield dividends down the road and that might be the reason I remember us setting all containers to expose 8000 as  each is in isolation by definition.